package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -358,13 +358,21 @@ def _seek(self, nframes):
 
         .. versionadded:: 0.9.0
         """
-        maxi_l = long(maxint)
-
-        framesize = long(self._dtype.itemsize)
-        seeksize = framesize * nframes
+        # On python 2, seek has issues with long int. This is solve in python 3
+        # where there is no longer a distinction between int and long int.
+        if six.PY2:
```

Comment:
This is not tested as it would require a very large file. Any idea to test that without such a large file is welcome. If somebody as a large file to test this at least once, it may be worth to try.